packages/components/src/Link.ts

```diff
@@ -0,0 +1,42 @@
+import { ForwardedRef, forwardRef } from 'react';
+import { h } from 'react-hyperscript-helpers';
+
+import { Clickable, ClickableProps } from './Clickable';
+import { useThemeFromContext } from './theme';
+
+export interface LinkProps extends ClickableProps {
+  baseColor?: (intensity?: number) => string;
+  variant?: 'light';
+}
+
+export const Link = forwardRef((props: LinkProps, ref: ForwardedRef<HTMLElement>) => {
+  const { baseColor: baseColorProp, children, disabled, hover, style, variant, ...otherProps } = props;
+
+  const { colors } = useThemeFromContext();
+  const baseColor = baseColorProp || colors.accent;
+
+  return h(
+    Clickable,
+    {
+      ref,
+      ...otherProps,
```

Comment:
In the non-TS version, wouldn't "otherProps" have potentially trumped other things because it was the second arg to _.merge? Or am I wrong about the ordering of the args with lodashfp?

Reply:
Right, but I'm worried that this is potentially a change in behavior… if someone wanted to override the hover and style props.

Reply:
That's still supported. The `style` prop overrides the default style. And the `hover` prop overrides the default `hover`, except when the button is disabled.

Reply:
Ah, I see. Thanks!